server-spi-private/src/main/java/org/keycloak/authorization/identity/Identity.java

```diff
@@ -66,4 +66,12 @@ default boolean hasRealmRole(String roleName) {
     default boolean hasClientRole(String clientId, String roleName) {
         return getAttributes().containsValue("kc.client." + clientId + ".roles", roleName);
     }
+
+    default boolean hasClientRole(String clientId, String... roleNames) {
```

Comment:
Please add a javadoc that this is has _any_ client role, so that it is not confused with has _all_ client roles